test/cmd/help.sh

```diff
@@ -50,6 +50,7 @@ os::cmd::expect_success_and_text 'oadm' 'Install Commands:'
 os::cmd::expect_success_and_text 'oadm ca' 'Manage certificates'
 os::cmd::expect_success_and_text 'openshift start kubernetes' 'Kubernetes server components'
 os::cmd::expect_success_and_text 'oc exec --help' '\[options\] \-\- COMMAND'
+os::cmd::expect_success_and_text 'oc rsh --help' '\[options\] \[COMMAND\]'
```

Comment:
Yeah, this one is just a short-term improvement but we need to do that. In fact, the templating system must be smarter about adding `[options]` to the usage section, today it just appends in case there are flags, but there are some cases (this and `exec` for example) where `[options]` for `oc` must come *before* the command being executed remotely.